ci/build_python.sh

```diff
@@ -9,13 +9,22 @@ export CMAKE_GENERATOR=Ninja
 
 rapids-print-env
 
+package_name="cuml"
+package_dir="python"
+
+version=$(rapids-generate-version)
+git_commit=$(git rev-parse HEAD)
+
+sed -i "s/__version__ = .*/__version__ = ${version}/g" ${package_dir}/${package_name}/__init__.py
+sed -i "s/__git_commit__ = .*/__git_commit__ = \"${git_commit}\"/g" ${package_dir}/${package_name}/__init__.py
```

Comment:
```suggestion
sed -i "/__version__/ s/= .*/= ${version}/g" ${package_dir}/${package_name}/__init__.py
sed -i "/__git_commit__/ s/= .*/= \"${git_commit}\"/g" ${package_dir}/${package_name}/__init__.py
```

These commands can be made slightly less verbose with the use of sed addresses: https://www.gnu.org/software/sed/manual/html_node/Addresses-overview.html#Addresses-overview

Reply:
That's clever, I like it. I've seen addresses but I never reach for them. Maybe I should!